ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/affordances/RDXRobotCollidable.java

```diff
@@ -97,7 +97,7 @@ public RDXRobotCollidable(FrameShape3DReadOnly shape,
       linkFrame = syncedLinkFrame;
 
       RigidBodyTransform collisionToLinkFrameTransform = new RigidBodyTransform();
-      collisionShapeFrame = new MutableReferenceFrame("collisionShapeFrame" + rigidBodyName, linkFrame);
+      collisionShapeFrame = new MutableReferenceFrame("collisionShapeFrame" + rigidBodyName + collidableIndex, linkFrame);
 
```

Comment:
need the index for robots like valkyrie which have multiple collidables per single rigid body